.github/workflows/build-and-push.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - arch: amd64
             toxenv: py27,py36,py37,py38,py39,py310,py311,pypy,pypy37,pypy38,pypy39
           - arch: arm64
-            toxenv: py38,py39,py311
+            toxenv: py38,py39,py311,py312
```

Comment:
It might make sense to add it also to amd64 which is faster.

Reply:
Yes. I did not notice the difference before actually running the CI.